README.md

```diff
@@ -4,4 +4,63 @@
 
 # Matryoshka
 
-This library provides generalized folds, unfolds, and traversals for fixpoint data structures in Scala.
+This library provides generalized folds, unfolds, and traversals for fixed point data structures in Scala.
+
+## External Resources
+
+- [Functional Programming with Bananas, Lenses, Envelopes and Barbed Wire](http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.41.125) – the iconic paper that collected a lot of this info for the first time
+- [Recursion Schemes: A Field Guide (Redux)](http://comonad.com/reader/2009/recursion-schemes/) – Ed Kmett’s summary (with links to Haskell code) of various folds and unfolds
+- [Unifying Structured Recursion Schemes](http://www.cs.ox.ac.uk/people/jeremy.gibbons/publications/urs.pdf) – a newer paper on how to generalize recursion schemes
+- [Efficient Nanopass Compilers using Cats and Matryoshka](https://github.com/sellout/recursion-scheme-talk/blob/master/nanopass-compiler-talk.org) – Greg Pfeil’s (Scala) talk on this library (and some other things)
+- [Fix Haskell (by eliminating recursion)](https://github.com/sellout/recursion-scheme-talk/blob/master/recursion-scheme-talk.org) – Greg Pfeil’s (Haskell) talk on recursion schemes
```

Comment:
I guess I would leave out "(Scala)" since it's kind of implied by the mention of "this library", and maybe rephrase as "Greg Pfeil’s talk on recursion schemes in Haskell." For that matter, also "Ed Kmett’s summary of various folds and unfolds, with links to Haskell code." Parentheticals hurt readability.